t/venom/pfservers/configurator/step2.yml

```diff
@@ -0,0 +1,159 @@
+name: step2 database-general-and-alerting 
+version: 2
+testcases:
+- name: step2_start_mariadb
+  steps:
+  - type: http
+    method: POST
+    url: '{{.pfserver_webadmin_url}}/api/v1/configurator/system_service/packetfence-mariadb/start'
+    ignore_verify_ssl: true
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: step2_create_root_account
```

Comment:
Title is wrong: `root` account is already created on DB.

When you called `secure_installation`, you are running https://mariadb.com/kb/en/mysql_secure_installation/.